vita3k/modules/SceAvPlayer/SceAvPlayer.cpp

```diff
@@ -258,11 +261,11 @@ EXPORT(int32_t, sceAvPlayerAddSource, SceUID player_handle, Ptr<const char> path
         }
         free(host.mem, buf);
         temp_file.close();
+        host.kernel.run_guest_function(player_info->file_manager.close_file.address(), { player_info->file_manager.user_data });
         if (fs::file_size(temp_file_path) != file_size) {
             LOG_ERROR("File is corrupted or incomplete: {}", temp_file_path.string());
-            return -1;
+            return RET_ERROR(-1);
```

Comment:
ret_error here no have sense, RET_ERROR is just for priint enum error in log, but here log is already writed